examples/ci_cd/gitlab-ci-cd/gitlab-provider.tf

```diff
@@ -1,23 +1,16 @@
-data "gitlab_group" "gitops-eks" {
-  full_path = "ssp-amazon-eks-terraform-group"
-}
 
-data "gitlab_projects" "ssp-amazon-eks" {
-  group_id          = data.gitlab_group.gitops-eks.id
-  simple            = true
-  search            = "ssp-amazon-eks"
-  include_subgroups = true
+# data source for an existing GitLab group
+data "gitlab_group" "gitops-eks" {
```

Comment:
consider maybe removing this data resource like you did for the project id?

Reply:
File deleted